client/src/routes/wind/LogRecord.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withStyles from 'isomorphic-style-loader/withStyles';
+import s from './css/Record.css';
```

Comment:
Why do you store css files separately? Store all files related to one component together

Reply:
to decrease the level of mess in files structure, as i have quite a few components and each of them has its own css file